src/schema/imported/theme.json

```diff
@@ -267,7 +267,7 @@
                   "$ref": "#/types/ThemeColor"
                 },
                 {
-                  "deprecated": "Please use <em>theme.colors.bookmark_text</em>, this alias will be removed in Firefox 69."
+                  "description": "This color property is an alias of 'bookmark_text'."
```

Comment:
Just verified, we didn't issue a deprecation warning for this specific property so we're safe here.